InstagramStories/Source/CollectionViewCell/IGStoryPreviewCell.swift

```diff
@@ -158,14 +167,30 @@ final class IGStoryPreviewCell: UICollectionViewCell, UIScrollViewDelegate {
         //Setting constraints for scrollview
         NSLayoutConstraint.activate([
             scrollview.leftAnchor.constraint(equalTo: contentView.leftAnchor),
-            scrollview.rightAnchor.constraint(equalTo: contentView.rightAnchor),
+            contentView.rightAnchor.constraint(equalTo: scrollview.rightAnchor),
             scrollview.topAnchor.constraint(equalTo: contentView.topAnchor),
-            scrollview.bottomAnchor.constraint(equalTo: contentView.bottomAnchor)])
+            contentView.bottomAnchor.constraint(equalTo: scrollview.bottomAnchor),
+            scrollview.widthAnchor.constraint(equalTo: contentView.widthAnchor, multiplier: 1.0),
+            scrollview.heightAnchor.constraint(equalTo: contentView.heightAnchor, multiplier: 1.0)
+            ])
+        NSLayoutConstraint.activate([
+            storyHeaderView.leftAnchor.constraint(equalTo: contentView.leftAnchor),
+            contentView.rightAnchor.constraint(equalTo: storyHeaderView.rightAnchor),
+            storyHeaderView.topAnchor.constraint(equalTo: contentView.topAnchor),
+            storyHeaderView.heightAnchor.constraint(equalToConstant: 80)
+            ])
     }
     private func createSnapView() -> UIImageView {
-        let snapView = UIImageView(frame: CGRect(x: snapViewXPos, y: 0, width: scrollview.frame.width, height: scrollview.frame.height))
+        let snapView = UIImageView()
+        snapView.translatesAutoresizingMaskIntoConstraints = false
         snapView.tag = snapIndex + snapViewTagIndicator
         scrollview.addSubview(snapView)
+        NSLayoutConstraint.activate([
```

Comment:
Write one line documentation for this constraint change

Reply:
done